### PR TITLE
[ACS-3549] add role dialog in manage version dialog

### PR DESCRIPTION
**Pull request by rbahirsheth**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Element must only use allowed ARIA attributes.

**What is the new behaviour?**
Fix the above issue by adding role dialog in manage version dialog

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: app/src/app/services/content-management.service.ts [hunk 1]
```diff
@@ -579,7 +579,7 @@ export class ContentManagementService {
         title: 'VERSION.DIALOG.TITLE'
       };
       this.newVersionUploaderService
-        .openUploadNewVersionDialog(newVersionUploaderDialogData, { width: '630px' })
+        .openUploadNewVersionDialog(newVersionUploaderDialogData, { width: '630px', role: 'dialog' })
```
Comment (MichalKinas) [inline app/src/app/services/content-management.service.ts:582]: Is this role required? According to material documentation -> MatDialog creates modal dialogs that implements the ARIA role="dialog" pattern by default. You can check it out here: https://material.angular.io/components/dialog/overview#accessibility

Comment (rbahirsheth) [reply app/src/app/services/content-management.service.ts:582]: You are right MatDialog adds ARIA role by default. But, somehow it is not creating the mat-dialog-container with role="dialog".
As you can see the below screenshot
image

Comment (MichalKinas) [reply app/src/app/services/content-management.service.ts:582]: Have you tried investigating why this happens in our dialog? I mean why the role isn't applied as it should be

Comment (rbahirsheth) [reply app/src/app/services/content-management.service.ts:582]: I tried but did not get the reason of this.

**Review by DenysVuika — APPROVED** [review]

**Pull request merged**